build-targets/ld/rival310-oem-boot.ld

```diff
@@ -4,4 +4,4 @@ MEMORY
     ram  (rw)     : ORIGIN = 0x20000000, LENGTH = 0x02800
 }
 
-INCLUDE stm32f1_common.ld
+INCLUDE ../build-targets/ld/stm32f1-common.ld
```

Comment:
I don't like this, but i couldn't find a work around. suggestions?

Reply:
Keep the original name, we can inject an include path to the linker.

Reply:
I tried that but was unsuccessful, care to tell me what the flag would be?

Reply:
Have you tried `-Lpath`?

Reply:
I did try, but i think i had a typo in the path.